### DISCUSSION TITLE
コメントのリンクを「...」メニューに移動しました

### DISCUSSION THREAD

**Discussion by ohakutsu** [increments/qiita-discussions#46] [アナウンス]
[コメントのリンクが日時クリックなのがわかりづらい Discussion #28](https://github.com/qiita-discussions/qiita-discussions/discussions/28)から立てられたIssueの[コメントのリンクが日時クリックなのがわかりづらい Issue #40](https://github.com/qiita-discussions/qiita-discussions/issues/40)の対応が完了しました！

リリースノートは[こちら](https://qiita.com/release-notes#%E3%83%89%E3%83%AD%E3%83%83%E3%83%97%E3%83%80%E3%82%A6%E3%83%B3%E3%83%A1%E3%83%8B%E3%83%A5%E3%83%BC%E3%81%8B%E3%82%89%E3%82%B3%E3%83%A1%E3%83%B3%E3%83%88%E3%81%AE%E3%83%AA%E3%83%B3%E3%82%AF%E3%82%92%E3%82%B3%E3%83%94%E3%83%BC%E3%81%A7%E3%81%8D%E3%82%8B%E3%82%88%E3%81%86%E3%81%AB%E3%81%AA%E3%82%8A%E3%81%BE%E3%81%97%E3%81%9F)です

Discussionを作成してくださった @uhooiさん ありがとうございました！

今回のようなDiscussionを通し、Issueが生まれ、解消されていくサイクルを今後もつくっていければと考えています
よろしくお願いします！

**Comment by kokosabu** [comment]
元のディスカッションを開いても、閉じたかどうかちょっと分からなそうでした。
ディスカッションを閉じるとか、アーカイブする機能ってGithub Discussionsにあるのでしょうかね。( すいません、機能を調べもせず発言しています。 )
機能がないのだったら、元のディスカッションにクローズした旨を残すなど、運用でカバーする必要があると思います。

**Reply by ohakutsu** [reply]
@kokosabu 
確かに閉じているか分からないですね...
現状、Discussionをクローズする機能がないので、運用面でカバーしていく必要がありそうです

ご意見ありがとうございます！

**Reply by blono** [reply]
今はこのリポジトリは Private ですが、Public になって Qiita ユーザー全員に Discussions を開放した場合どの程度の量の Issue や Disussion が発生するかがふと心配になりました（果たして、管理が行き届くのだろうかという... 😱

**Reply by kskwtnk** [reply]
@kokosabu 
ご意見ありがとうございます！
アーカイブ、クローズといった機能が現状のGitHub Discussionsには存在していないようです……
GitHub本家のDiscussions内でもたくさんvoteやリアクションがついているので、もしかしたら実装されるかもしれませんが今は運用でカバーしていこうと思います！
https://github.com/github/feedback/discussions/3097

方針を決めたらアナウンスを展開します！

@blono 
仰るとおりですね……publicにした際にどれくらいのDiscussions量になるのか、長く続いた場合の管理がどうなるのか
現状私たちとしても読めていないのですが、こちらもテストしていきたいと思っています

クローズドのままで人数を増やす、一旦パブリックにしつつも「テスト期間のため全てに回答できるとは限りません」といったコミュニケーションをとる、など方法はいくらか考えられます

こちらも本運用に向けて考えていくポイントなので、またみなさんにもアナウンスできればと思っています！

**Comment by uhooi** [comment]
うおおおお
迅速なご対応ありがとうございます… 🙏

**Reply by kskwtnk** [reply]
@uhooi 

こちらこそ、Discussion作成ありがとうございました！
